cybershell/memory.py

```
@@ -0,0 +1,74 @@
+    def __init__(self):
+        self.items: List[MemoryItem] = []
+        self.vec = SimpleVectorizer()
+        self._matrix: List[List[float]] = []
+
+    def add(self, item: MemoryItem):
+        self.items.append(item); self._reindex()
+
+    def bulk_add(self, items: List[MemoryItem]):
+        self.items.extend(items); self._reindex()
+
+    def _reindex(self):
```

Ruff (0.12.2)

Missing return type annotation for private function `_reindex`

Add return type annotation: `None`

(ANN202)